### PR TITLE
Update data-EgammaAnalysis-ElectronTools.spec

### PR DESCRIPTION
**Pull request by mrodozov**
Description in : https://github.com/cms-data/EgammaAnalysis-ElectronTools/pull/4

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

**Comment by smuzaffar** [comment]
hold, we have to update the tag once https://github.com/cms-data/EgammaAnalysis-ElectronTools/pull/6 is merged

File: data-EgammaAnalysis-ElectronTools.spec [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### RPM cms data-EgammaAnalysis-ElectronTools V00-01-01
+### RPM cms data-EgammaAnalysis-ElectronTools V00-01-02
```
Comment (slava77) [inline data-EgammaAnalysis-ElectronTools.spec]: this should now change to -03, IIUC

**Comment by mrodozov** [comment]
please test

**Comment by slava77** [comment]
> Reco comparison results: 5 differences found in the comparisons

as expected, these are the same changes RunJetHT2016E reMINIAOD wf 136.7611 as in the earlier tests.
This confirms that this PR is OK for data.
The other changes were specific to MC.

I ran 1325.5 (and a higher stat test of 136.7611) locally and see that test passed OK:
- changes are only in userFloats of slimmedElectrons and slimmedPhotons 
- there are small differences in more userFloats in 1325.5 , corresponding to the added smearing

I think that this is good to go.

**Comment by fabiocos** [comment]
+externals

@slava77 thanks a lot for the checks

**Comment by fabiocos** [comment]
+1

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**